### ISSUE TITLE
Running from .bpmn files

### ISSUE THREAD

**Issue by PetrMixayloff** [viewflow/viewflow#284]
Is it any way for running workflow from .bpmn files creating in Comunda or may be any other software for creating bpmn diagrams?

**Comment by kmmbvnr** [comment]
BPMN diagrams itself has not enough information to run the process. Only task names/types and task connections

**Comment by PetrMixayloff** [comment]
> BPMN diagrams itself has not enough information to run the process. Only task names/types and task connections
I mean xml files with .bpmn extensions, for example, this I created with comunda:
`<bpmn:process id="Process_07f1h2r" isExecutable="true">
    <bpmn:startEvent id="StartEvent_1" name="Create research" camunda:formKey="research">
      <bpmn:extensionElements>
        <camunda:properties>
          <camunda:property />
        </camunda:properties>
        <camunda:formData>
          <camunda:formField id="FormField_3cm7nq2" label="name" type="string" defaultValue="111" />
        </camunda:formData>
      </bpmn:extensionElements>
      <bpmn:outgoing>Flow_0cnzusx</bpmn:outgoing>
    </bpmn:startEvent>
    <bpmn:sequenceFlow id="Flow_0cnzusx" sourceRef="StartEvent_1" targetRef="Activity_03crika" />
    <bpmn:userTask id="Activity_03crika" name="Make research" camunda:assignee="user1">
      <bpmn:incoming>Flow_0cnzusx</bpmn:incoming>
      <bpmn:outgoing>Flow_1418ls1</bpmn:outgoing>
    </bpmn:userTask>
    <bpmn:endEvent id="Event_18h11bt" name="end">
      <bpmn:incoming>Flow_1418ls1</bpmn:incoming>
    </bpmn:endEvent>
    <bpmn:sequenceFlow id="Flow_1418ls1" sourceRef="Activity_03crika" targetRef="Event_18h11bt" />
  </bpmn:process>`
Here can be input/output parameters, properties, scripts, is it really not enough to execute workflow? There is one project on github, SpiffWorkflow, I think you must know it. It is possible to execute workflow from xml and json there.

**Comment by kmmbvnr** [comment]
Yep, camuunda has they own extensions of BPMN formats. Other tools has own export formats also. BPNM itself is only about task names and links between.
